src/NexusMods.MnemonicDB/Storage/DatomStore.cs

```diff
@@ -308,7 +309,7 @@ private StoreResult Log(IInternalTxFunctionImpl pendingTransaction)
         
         return new StoreResult
         {
-            AssignedTxId = _thisTx,
+            AssignedTxId = _asOfTx,
```

Comment:
This is the source of the bug, when we had an empty transaction (and we do anytime we reboot the app and the schema doesn't update), then we were incrementing the TxId. The internal functions increment the `_asOfTx` so we no longer need to use `_thisTx`.